<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartPage</class>
 <widget class="QWidget" name="StartPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="MainButton" name="btnRank">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>424</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::StrongFocus</enum>
   </property>
   <property name="text">
    <string>排行榜</string>
   </property>
  </widget>
  <widget class="MainButton" name="btnLevel">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>244</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string>闯关模式</string>
   </property>
  </widget>
  <widget class="MainButton" name="btnCustom">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>334</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string>自定义模式</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainButton</class>
   <extends>QPushButton</extends>
   <header location="global">mainbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
